Templates/ScriptCanvasNode/Template/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+ly_add_target(
+    NAME ${Name}.Static STATIC
+    NAMESPACE ScriptCanvasNodes
+    FILES_CMAKE
+        ${Name}_files.cmake
+    INCLUDE_DIRECTORIES
+        PRIVATE
+            Source
+            .
+        PUBLIC
+            .
+            Include
```

Comment:
Folder structure wise, I think the following should occur.
The `.` shouldn't be added as an include directory.
Having the `.` directory allows the current CMakeLists.txt directory to be polluted with header files.

Instead the `${Name}.h` file can go into the Include directory.
Now for the `${Name}.cpp` file, I recommend placing it either in 1. the Source directory for purity of separate of public API/private implementation or 2. the "Include" directory for convenience of being next to the `${Name}.h` header